src/Hammer.cs

```diff
@@ -9,6 +9,7 @@ public class Hammer : Tool
         private double total;
 
         public double Average => total / done;
+        private readonly ChartLabels _chartLabels = new ChartLabels();
```

Comment:
since the default values are being removed, make sure they get set explicitly here